### PR TITLE
service/rtc: Fixing repeat initialization of connection

### PR DESCRIPTION
**Pull request by agnivade**
MultiConn was being incorrectly initialized inside a loop.
This created repeated listener goroutines of the same connection.

### CODE REVIEW

**Review by streamer45 — DISMISSED** [review]
Thanks for spotting this :raised_hands:  I'll run a load-test before merging just to make extra sure all is good since unfortunately our initial perf tests were conducted with the bogus implementation although I don't expect to see substantial differences.

**Comment by streamer45** [comment]
Load-tests look good. The bug could have easily had a much worse impact especially since the "leaked" readers would still receive data from the underlying connection but never deliver it. Fortunately they would block on channel send so at most one datagram would be lost per reader which is likely the reason this passed unnoticed for so long.

**Review by cpoile — APPROVED** [review]
Thanks @agnivade!

**Comment by agnivade** [comment]
@streamer45 - It seems like it requires commits to be signed to merge a PR. Do you want me to squash and force-push?

**Comment by agnivade** [comment]
@cpoile - can you please approve once more? Thanks!

**Review by cpoile — APPROVED** [review]
👍

**Pull request merged**
